### ISSUE TITLE
Heading is always displayed

### ISSUE THREAD

**Issue by EnderMelody** [Oth3r/DirectionHUD-F#9]
minecraft 1.20.1
mod version: 1.3.9

i wanted to use this mod only for the location/player tracking but even with all the display options off the heading is still shown.

**Comment by Oth3r** [comment]
You can do '/hud toggle' to turn off the HUD, can you give more details?

**Comment by EnderMelody** [comment]
i have a different mod to display the compass heading, but i would like to use this mod to track players/locations. i turned off all other options in the settings as i dont need them, however the N/E/S/W is still displayed over my toolbar. the command you gave will work for this, however the bug is still present.

**Comment by Oth3r** [comment]
hmm alright, i gave it a go, and it seems to go away just fine, are you disabling it in the config or in game?
![07 13 23-15 55 59](https://github.com/Oth3r/DirectionHUD-Fabric/assets/68134921/39e5e680-36a5-4fc8-b067-6f6248aeb95c)

**Comment by EnderMelody** [comment]
thats odd, i changed it in the mod config ui as well as in the config file itself. ill try the chat ui to make sure.

**Comment by Oth3r** [comment]
ahh that makes sense, the config file/ui is for default configuration, (for loading new worlds, players, ect), if you already loaded before changing the default config, you have to hit the reset button or manually set it

ill try to make that more clear in future versions of the mod

**Comment by EnderMelody** [comment]
yup, can confirm the config doesnt work but the chat config works fine.
also the config was changing post world creation without any other inputs such as the reset button so idk if that is the cause of the problem. other than that one config but everything seems to be working now, thanks!
